src/main/scala/kamon/instrumentation/http/HttpClientInstrumentation.scala

```diff
@@ -195,10 +195,7 @@ object HttpClientInstrumentation {
       } recover {
         case t: Throwable =>
           _log.warn("Failed to create an HTTP Operation Name Generator, falling back to the default operation name", t)
-
-          new HttpOperationNameGenerator {
-            override def name(request: HttpMessage.Request): Option[String] = Some(defaultOperationName)
-          }
+          new HttpOperationNameGenerator.Static(defaultOperationName)
```

Comment:
refactored this just to keep the universe in balance as the recover/fail behaviour for the client/server side name-generators are the same